incubator/acs-engine-autoscaler/README.md

```diff
@@ -0,0 +1,119 @@
+
+To verify that acs-engine-autoscaler is running as expected, run:
+  kubectl logs $(kubectl --namespace=default get pods -l "app=olfactory-bunny-acs-engine-autoscaler" -o jsonpath="{.items[0].metadata.name}")
+
+$ kubectl --namespace=default get pods -l "app=olfactory-bunny-acs-engine-autoscaler"
+
+NAME                                                     READY     STATUS    RESTARTS   AGE
+olfactory-bunny-acs-engine-autoscaler-1715934483-c673v   1/1       Running   0          10s
+
+$ kubectl logs $(kubectl --namespace=default get pods -l "app=olfactory-bunny-acs-engine-autoscaler" -o jsonpath="{.items[0].metadata.name}")
+
+2017-06-11 23:20:59,352 - autoscaler.cluster - DEBUG - Using kube service account
+2017-06-11 23:20:59,352 - autoscaler.cluster - INFO - ++++ Running Scaling Loop ++++++
+2017-06-11 23:20:59,421 - autoscaler.cluster - INFO - Pods to schedule: 0
+2017-06-11 23:20:59,421 - autoscaler.cluster - INFO - ++++ Scaling Up Begins ++++++
+2017-06-11 23:20:59,421 - autoscaler.cluster - INFO - Nodes: 1
+2017-06-11 23:20:59,421 - autoscaler.cluster - INFO - To schedule: 0
+2017-06-11 23:20:59,421 - autoscaler.cluster - INFO - Pending pods: 0
+2017-06-11 23:20:59,422 - autoscaler.cluster - INFO - ++++ Scaling Up Ends ++++++
+2017-06-11 23:20:59,422 - autoscaler.cluster - INFO - ++++ Maintenance Begins ++++++
+2017-06-11 23:20:59,422 - autoscaler.engine_scaler - INFO - ++++ Maintaining Nodes ++++++
+2017-06-11 23:20:59,423 - autoscaler.engine_scaler - INFO - node: k8s-agentpool1-29744472-4                                                   state: under-utilized-undrainable
+2017-06-11 23:20:59,423 - autoscaler.cluster - INFO - ++++ Maintenance Ends ++++++
+...
+```
+
+## Uninstalling the Chart
+
+To uninstall/delete the last deployment:
+
+```console
+$ helm ls
+$ helm delete [last deployment]
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following tables lists the configurable parameters of the acs-engine-autoscaler chart and their default values.
+
+Parameter | Description | Default
+--- | --- | ---
+`resourcegroup`| Name of the resource group containing the cluster | None. You *must* supply one.
+`azurespappid`| An Azure service principal id | None. You *must* supply one.
+`azurespsecret`| An Azure service principal secret | None. You *must* supply one.
+`azuresptenantid`| An Azure service principal tenant id | None. You *must* supply one.
+`kubeconfigprivatekey`| The key passed to the `kubeConfigPrivateKey` parameter in your `azuredeploy.parameters.json` generated with `acs-engine` | None. You *must* supply one.
+`clientprivatekey`| The key passed to the `clientPrivateKey` parameter in your `azuredeploy.parameters.json` generated with `acs-engine` | None. You *must* supply one.
+`acsdeployment`| [OPTIONAL] The name of the deployment used to deploy the kubernetes cluster initially. If not provided, it is defaulted to `azuredeploy` | None.
```

Comment:
This is actually the name that was used to deploy the cluster generated with acs-engine via `az group deployment create ...` .